### PR TITLE
Added reset icon after activate chat window

### LINKED ISSUE #3139: Icon blinking after reading message
##### Brief Description

OS: Arch Linux
qTox version: v1.2.2-444-g0a2f541
Commit hash:  0a2f54194569e626ef8501c610647219887505d4
toxcore: 0.0.0
Qt:  5.6.0 
…

Reproducible: Always
##### Steps to reproduce
1. Go to multi window mode
2. Collapse chat window
3. Get message
4. Expand chat window
##### Observed Behavior

Contacts icon will blink
##### Expected Behavior

Stop blinking icon

### PR DESCRIPTION
**Pull request by Diadlo**
Fix #3139.

### CODE REVIEW

File: src/widget/widget.h [hunk 1]
```diff
@@ -104,6 +104,7 @@ class Widget final : public QMainWindow
     void searchCircle(CircleWidget* circleWidget);
     void searchItem(GenericChatItemWidget* chatItem, GenericChatItemWidget::ItemType type);
     bool groupsVisible() const;
+    void resetIcon();
```
Comment (zetok) [inline src/widget/widget.h:109]: Could you add a comment explaining what this function does and why?

Comment (Diadlo) [inline src/widget/widget.h:109]: @zetok to reset icon status :)

I can't set value of `eventIcon` and `eventFlag` without it.
I need it to stop blinking

Comment (zetok) [inline src/widget/widget.h:109]: um, yes, but above the code

Comment (Diadlo) [inline src/widget/widget.h:109]: @zetok Like now?

**Pull request merged**